src/EventTickets/ListTable/Columns/SalesCountColumn.php

```diff
@@ -21,7 +21,7 @@ class SalesCountColumn extends ModelColumn
      */
     public static function getId(): string
     {
-        return 'sales-count';
+        return 'salesCount';
```

Comment:
As a note, the IDs don't have to be updated. We can use array syntax in JavaScript when accessing this key.

Reply:
This has only been renamed as we are using camelCase for others.